packages/core/src/babel-plugin-redwood-cell.ts

```diff
@@ -1,33 +1,43 @@
-// This is supposed to wrap a file that's has a suffix of `Cell` in Redwood's `withCell` higher order component.
-// The HOC is responsible for the lifecycle methods during a graphQL query.
-// The end result of this plugin is something like:
+import type { PluginObj, types } from '@babel/core'
+
+// This wraps a file that has a suffix of `Cell` in Redwood's `withCell` higher
+// order component. The HOC deals with the lifecycle methods during a GraphQL query.
+//
 // ```js
 // import { withCell } from '@redwoodjs/web'
+// <YOUR CODE>
 // export default withCell({ QUERY, Loading, Succes, Failure, Empty, beforeQuery, afterQuery })
 // ```
-module.exports = function ({ types: t }) {
-  // These are the expected named exports from a Cell file.
-  const EXPECTED_EXPORTS_FROM_CELL = [
-    'QUERY',
-    'Loading',
-    'Success',
-    'Failure',
-    'Empty',
-    'beforeQuery',
-    'afterQuery',
-  ]
 
+// A cell can export the declarations below.
+const EXPECTED_EXPORTS_FROM_CELL = [
+  'beforeQuery',
+  'QUERY',
+  'afterQuery',
+  'Loading',
+  'Success',
+  'Failure',
+  'Empty',
+]
+
+export default function ({ types: t }: { types: typeof types }): PluginObj {
   // This array will
   // - collect exports from the Cell file during ExportNamedDeclaration
   // - collected exports will then be passed to `withCell`
   // - be cleared after Program exit to prepare for the next file
-  let exportNames = []
+  let exportNames: string[] = []
+  let hasDefaultExport = false
 
   return {
+    name: 'babel-plugin-redwood-cell',
     visitor: {
+      ExportDefaultDeclaration() {
+        // This is not a cell since it exports a default.
```

Comment:
Oh, I forgot to move (and update) this comment to the new place where we check for both presence of default export and the absence of QUERY export.

Reply:
Thanks for the reminder! I'll move it to the right place. Btw, thanks for the work on this PR!